low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -350,8 +350,7 @@ def __init__(self, args):
         from cortx.sspl.bin.sspl_constants import PRODUCT_FAMILY
 
         self.SSPL_CONFIGURED="/var/%s/sspl/sspl-configured" % (PRODUCT_FAMILY)
-        self.services = ["rabbitmq-server", "sspl-ll"]
-        Service('dbus').process('start', 'sspl-ll.service')
```

Comment:
If not, can you let us know from where sspl-ll.service is started?

Reply:
rabbitmq-server should be started durng dev deploy, until message bus integration.
sspl-ll service start will have to be manual & documented i believe. Ideally with integrated Cortx stack, sspl will be started by HA through PCS interfaces only(using systemd resource agent).

Reply:
Yes, there was no SSPL start in previous `sspl_setup`. Also there was bug here. SSPL  start was in `CheckCmd` which gets executed in `ExecStartPre`. So when starting SSPL, it checks SSPL is already running or not.